activiti-engine/src/main/java/org/activiti/engine/repository/DiagramLayout.java

```diff
@@ -1,74 +0,0 @@
-/* Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
@daisuke-yoshimoto do we need this class in the image-generator project? I'm not sure but as far as I remember there is a structure that keeps track of the coordinates of the activities in the diagram, is this the class that stores that information?